llmfoundry/models/mpt/modeling_mpt.py

```diff
@@ -1121,6 +1126,7 @@ def forward(
         input_ids: Optional[torch.LongTensor] = None,
         past_key_values: Optional[list[tuple[torch.FloatTensor]]] = None,
         attention_mask: Optional[torch.ByteTensor] = None,
+        position_ids: Optional[torch.LongTensor] = None,
```

Comment:
any particular reason you put this in the middle of the args? it'd be better to put it at the end probably so it doesn't break any existing calls to this function

Reply:
fixed - and added test. ptal!